tests/test_fyle/test_tasks.py

```
@@ -1248,3 +1251,243 @@ def test_cleanup_scheduled_task_not_found(
+        'report_id': None
+    }
+
+    _handle_expense_ejected_from_report(expense, expense_data, workspace)
+
+    accounting_export.refresh_from_db()
+
+    assert accounting_export.expenses.count() == initial_count
+    assert expense in accounting_export.expenses.all()
+
+
+def test_handle_expense_report_change_ejected_expense_not_found(db, mocker):
```

Ruff (0.14.0)

Unused function argument: `db`

(ARG001)